__init__.py

```
@@ -112,6 +96,25 @@
+
+            if self.wmctrl:
+                # TODO - prompt user if they want to switch to open window
+                switch = True
+                if switch:
+                    win = self.match_window(app)
+                    window_id = win[0][0] if win else None
+                    self.switch_window(window_id, self.wmctrl)
+                    return True
+            if not switch:
+                # TODO - prompt user if they want to launch a new instance
+                if not launch:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)